controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -680,4 +680,22 @@
      * @return Completable future that, upon completion, holds the epoch in which the segment was sealed.
      */
     CompletableFuture<Integer> getSegmentSealedEpoch(long segmentId);
+
+    /**
+     * Method to compare streamcuts to check if streamcut1 is strictly ahead of streamcut2. 
+     * @param cut1 streamcut to check
+     * @param cut2 streamcut to check against. 
+     *
+     * @return CompletableFuture which, upon completion, will indicate if the streamcut1 is strictly ahead of streamcut2.
+     */
+    CompletableFuture<Boolean> streamCutStrictlyGreaterThan(Map<Long, Long> cut1, Map<Long, Long> cut2);
```

Comment:
method name should be `isStreamCutStrictlyGreaterThan` since it returns a boolean.